internal/marketplaces/namespaces/validation.go

```diff
@@ -51,7 +51,7 @@ func ValidateNamespacedNameRules(name string, subscriptionID uuid.UUID) error {
 //
 //	subscription ID.
 //
-// B) If the subscription ID from the DB is null, the subscriptionID parameter
+// B) If the subscription ID from the DB is null, the subscriptionID parameterß
```

Comment:
nein!

Reply:
Will fix this in my next PR.... I think this is the result of me hitting Option instead of Command on my keyboard :)